keyboards/ryanskidmore/rskeys100/matrix.c

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
Please convert this to the lite custom matrix. 

https://docs.qmk.fm/#/custom_matrix?id=lite

You can use the ergodox EZ or moonlander as an example of how to do this.

Reply:
I've migrated to the `lite` implementation now - thanks for the heads up! The code looks a lot cleaner with a bunch of the boilerplate removed.